bin/sl-run.js

```diff
@@ -11,15 +11,42 @@ process.on('disconnect', function() {
 var config = require('../lib/config'); // May exit, depending on argv
 var log = config.logger;
 
-if(!config.profile) {
-  if(config.isMaster) {
-    log.error('supervisor running without StrongOps (unprofiled)');
-  }
-} else {
-  require('strong-agent').profile(undefined, undefined, {
-    quiet: config.isWorker, // Quiet in worker, to avoid repeated log messages
-    logger: config.logger,
-  });
+var agent = require('../lib/agent');
+var options = {
```

Comment:
It's really really minor, but could this be `agentOptions` to make the association more clear?